public-ui/src/app/views/pm/pm.component.html

```diff
@@ -1,7 +1,72 @@
-  </a>
+<div class="row">
+  <div class="col-xs-12 col-sm-12 col-md-9">
 
-</p>
+    <div class="page-header">
+      <div>
+        <h2> {{title}} </h2>
+      </div>
+    </div>
+
+    <div class="row page-align">
+      <div *ngFor="let item of conceptGroups" class="button-item">
+        <button (click)="showMeasurement(item, item.concepts[0])" [ngClass]="{active:selectedGroup === item}"
+                class="btn btn-primary"> {{item.groupName}} </button>
+      </div>
+    </div>
+
+    <div *ngIf="!loading()" class="measurement white-card-box ">
+      <div class="row box-title">
+        <div class="col-md-4">
+          <h5 class="box-title"> {{selectedGroup.groupName}}</h5>
+        </div>
+        <div *ngIf="selectedGroup.concepts.length > 1" class="col-md">
+          <button *ngFor="let c of selectedGroup.concepts" [ngClass]="{active:selectedConcept === c}" (click)="showMeasurement(selectedGroup, c)" class="btn btn-link">{{c.conceptName}} </button>
+        </div>
+      </div>
+
+      <div class="row" *ngIf="selectedConcept.analyses && selectedConcept.analyses.genderAnalysis" >
+        <!-- Loop through the genders we have results for showing the graph -->
+        <div *ngFor="let gender of selectedConcept.analyses.genderAnalysis.results" class="col-md-4 col-xs-12 chart-wrapper">
+
+
+              <app-chart *ngIf="gender.stratum2 === dbc.MALE_GENDER_ID && selectedConcept.analyses &&
+                         selectedConcept.analyses.measurementValueMaleAnalysis"
+                         [chartTitle]="maleGenderChartTitle" [chartType]="selectedConcept.chartType"
+                         [analysis]="selectedConcept.analyses.measurementValueMaleAnalysis"></app-chart>
+
+              <app-chart *ngIf="gender.stratum2 === dbc.FEMALE_GENDER_ID && selectedConcept.analyses &&
+                         selectedConcept.analyses.measurementValueFemaleAnalysis"
+                         [chartTitle]="femaleGenderChartTitle" [chartType]="selectedConcept.chartType"
+                         [analysis]="selectedConcept.analyses.measurementValueFemaleAnalysis"></app-chart>
+
+              <app-chart *ngIf="gender.stratum2 === dbc.OTHER_GENDER_ID && selectedConcept.analyses &&
+                         selectedConcept.analyses.measurementValueOtherGenderAnalysis"
+                         [chartTitle]="otherGenderChartTitle" [chartType]="selectedConcept.chartType"
+                         [analysis]="selectedConcept.analyses.measurementValueOtherGenderAnalysis"></app-chart>
+
+        </div>
+
+        <!-- show Age totals -->
+        <div *ngIf="selectedConcept.analyses && selectedConcept.analyses.ageAnalysis" class="col-md-4 col-xs-12 chart-wrapper" >
+          <app-chart [analysis]="selectedConcept.analyses.ageAnalysis" [pointWidth]="20" [backgroundColor]="backgroundColor"> </app-chart>
+        </div>
+
+        <!-- Todo wait on design for further dev -->
+
+      </div>
+    </div>
+
+    <!-- Don't want to deploy this link
```

Comment:
remove? This doesn't seem like it should be in the code

Reply:
I can remove this. It is just handy and this screen will be redesigned.